oracle_hardhat/README.md

```diff
@@ -0,0 +1,3 @@
+## BTC Price Oracle (Hardhat network)
```

Comment:
Hm on main we already have oracle_hardhat service in packages - accidental commit?